README.md

```diff
@@ -83,36 +89,9 @@ To enable the sharing functionality on Vercel, you need to configure the followi
 - `VITE_DISCORD_CHANNEL_ID` - Discord channel ID for embedded chat
 - `VITE_DISCORD_SERVER_ID` - Discord server ID
 - `VITE_DISCORD_INVITE_URL` - Discord invite URL (defaults to https://authzed.com/discord)
+- `VITE_AUTHZED_DEVELOPER_GATEWAY_ENDPOINT` - ????
```

Comment:
???

Reply:
Not needed anymore; we don't use a gateway any further

Reply:
removed then